teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
```

Comment:
- [ ] You should have a `CapitalizeDecorator` class, it should look similar to this 👇 

```ruby
class CapitalizeDecorator < Decorator
    # Capitalize the first letter of the given name.
    def correct_name
      @nameable.correct_name.capitalize
    end
end
```

- [ ] You should also have the `TrimmerDecorator` class, it should look similar to:

```ruby
class TrimmerDecorator < Decorator
    # The name has a maximum of 10 characters. If it's longer it should trim the word.
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
end
```

Reply:
Thanks , this has been resolved

Reply:
This changes has been done well.